src/components/PortfolioCard.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+import { layoutMixins } from '@/styles/layoutMixins';
+
+import { AssetIcon } from './AssetIcon';
+
+type PortfolioCardProps = {
+  assetName: string;
+  assetId: string;
+  actionSlot: React.ReactNode;
+
+  detailLabel: string;
+  detailValue: React.ReactNode;
+};
+
+export const PortfolioCard = ({
+  assetId,
+  assetName,
+  actionSlot,
+  detailLabel,
+  detailValue,
+}: PortfolioCardProps) => {
+  return (
+    <$PortfolioCard>
+      <$MarketRow>
+        <AssetIcon symbol={assetId} />
+        {assetName}
+      </$MarketRow>
+      <$MarginRow>
+        <$MarginLabel>{detailLabel}</$MarginLabel>
+        <$MarginValue>{detailValue}</$MarginValue>
+      </$MarginRow>
+      <$ActionRow>{actionSlot}</$ActionRow>
+    </$PortfolioCard>
+  );
+};
+
+const $PortfolioCard = styled.div`
+  ${layoutMixins.flexColumn}
+  width: 14rem;
+  min-width: 14rem;
+  background-color: var(--color-layer-3);
+  overflow: hidden;
+  padding: 0.75rem 0;
+  padding-bottom: 0.5rem;
+  border-radius: 0.625rem;
```

Comment:
Very similar to the UnopenedIsolatedPositionCard, could consolidate styling as a new mixin object?

Reply:
Ah disregard, I see that you consolidated already